ova/Libraries/provision-opendistro.sh

```diff
@@ -0,0 +1,293 @@
+            logger -ne $char
+            sleep 10
+        done
+
+        cd /usr/share/elasticsearch/plugins/opendistro_security/tools/ 
+        ./securityadmin.sh -cd ../securityconfig/ -nhnv -cacert /etc/elasticsearch/certs/root-ca.pem -cert /etc/elasticsearch/certs/admin.pem -key /etc/elasticsearch/certs/admin.key 
+
+        logger "Done"
+    fi
+}
+
+## Filebeat
+installFilebeat() {
+    logger "Installing Filebeat..."
+
+    $sys_type install filebeat-"${ELK_VERSION}" -y -q  
+    if [ "$?" != 0 ]; then
+        logger "Error: Filebeat installation failed"
+        exit 1;
+    else
+        WAZUH_MAJOR="$(echo ${WAZUH_VERSION} | head -c 1)"
+
+        curl -so /etc/filebeat/filebeat.yml ${resources_url}/resources/open-distro/filebeat/7.x/filebeat_all_in_one.yml --max-time 300  
+        curl -so /etc/filebeat/wazuh-template.json https://raw.githubusercontent.com/wazuh/wazuh/master/extensions/elasticsearch/7.x/wazuh-template.json --max-time 300 
+        chmod go+r /etc/filebeat/wazuh-template.json 
+        
+        curl -s https://packages.wazuh.com/3.x/filebeat/wazuh-filebeat-0.1.tar.gz --max-time 300 | tar -xvz -C /usr/share/filebeat/module 
+        mkdir -p /etc/filebeat/certs 
+        cp /etc/elasticsearch/certs/root-ca.pem /etc/filebeat/certs/ 
+        mv /etc/elasticsearch/certs/filebeat.key /etc/filebeat/certs/ 
+        mv /etc/elasticsearch/certs/filebeat.pem /etc/filebeat/certs/ 
+        # Start Filebeat
+        startService "filebeat"
+        logger "Done"
+    fi
+}
+
+## Kibana
+installKibana() {
+    WAZUH_MAJOR="$(echo ${WAZUH_VERSION} | head -c 1)"
+    logger "Installing Open Distro for Kibana..."
+    $sys_type install opendistroforelasticsearch-kibana-${OPENDISTRO_VERSION} -y -q 
+    if [ "$?" != 0 ]; then
+        logger "Error: Kibana installation failed"
+        exit 1;
+    else
+        curl -so /etc/kibana/kibana.yml ${resources_url}/resources/open-distro/kibana/7.x/kibana_all_in_one.yml --max-time 300 
+        echo "telemetry.enabled: false" >> /etc/kibana/kibana.yml
+        cd /usr/share/kibana 
+
+        if [ "${STATUS_PACKAGES}" = "prod" ]; then
+            sudo -u kibana /usr/share/kibana/bin/kibana-plugin install https://packages.wazuh.com/${WAZUH_MAJOR}.x/ui/kibana/wazuh_kibana-${WAZUH_VERSION}_${ELK_VERSION}.zip 
+        elif [ "${STATUS_PACKAGES}" = "dev" ]; then
+            sudo -u kibana /usr/share/kibana/bin/kibana-plugin install https://packages-dev.wazuh.com/pre-release/ui/kibana/wazuh_kibana-${WAZUH_VERSION}_${ELK_VERSION}-1.zip 
+        fi
+                
+        if [ "$?" != 0 ]
+        then
+            logger "Error: Wazuh Kibana plugin could not be installed."
+            exit 1;
+        fi
+        mkdir -p /etc/kibana/certs 
+        mv /etc/elasticsearch/certs/kibana.key /etc/kibana/certs/ 
+        mv /etc/elasticsearch/certs/kibana.pem /etc/kibana/certs/ 
```

Comment:
```suggestion
        mv /etc/elasticsearch/certs/{kibana.key,kibana.pem} /etc/kibana/certs/ 
```

Reply:
Done in 599737656bcaaa119a2c30c76c1760fa0590643a